node/primitives/src/lib.rs

```diff
@@ -41,6 +41,8 @@ use std::pin::Pin;
 
 pub use sp_core::traits::SpawnNamed;
 
+pub mod approval;
```

Comment:
should this go into `v1`? It's not clear to me what should go directly here and what is versioned.